etc/docker/nb2kg/start-nb2kg.sh

```diff
@@ -2,32 +2,17 @@
 
 export NB_PORT=${NB_PORT:-8888}
 export GATEWAY_HOST=${GATEWAY_HOST:-localhost}
-export KG_URL=${KG_URL:-http://${GATEWAY_HOST}:8888}
+export KG_URL=${KG_URL:-http://${GATEWAY_HOST}:${NB_PORT}}
 export KG_HTTP_USER=${KG_HTTP_USER:-jovyan}
 export KG_REQUEST_TIMEOUT=${KG_REQUEST_TIMEOUT:-30}
 export KERNEL_USERNAME=${KG_HTTP_USER}
 
+
 echo "Starting nb2kg against gateway: " ${KG_URL}
 echo "Nootbook port: " ${NB_PORT}
 echo "Kernel user: " ${KERNEL_USERNAME}
 
-CMD=${1:-"notebook"}
-if [[ "${CMD}" == "lab" ]];
-then
-	jupyter serverextension enable --py jupyterlab --sys-prefix
-elif [[ "${CMD}" != "notebook" ]];
-then
-	echo ""
-	echo "usage: <docker run arguments> [notebook | lab]"
-	echo "Entering shell..."
-	/bin/bash
-	exit 0
-fi
+echo "${@: -1}"
+
```

Comment:
Here's the code relative to the previous comment.  May need to add something equivalent.